lib/modules/commentNavigator.js

```diff
@@ -89,7 +88,7 @@ modules['commentNavigator'] = {
 						case 'images':
 							thisEle.setAttribute('title', 'Navigate through comments with images');
 							break;
-						case 'images':
+						case 'videos':
```

Comment:
@erikdesjardins I have to ask - I tried using IntelliJ with RES and I can't because it seems to not properly obey / update symlinked files when I save the main files (e.g. lib/modules/foo.js) - so I don't see the updates w hen I refresh the extension.  How are you addressing this problem, or is it unique to me and my setup / OS?  I reverted to Sublime for this reason instead of starting to take advantage of IntelliJ!

Are you on OSX or Windows?